File: java/src/main/java/br/com/vps/consulting/b2b/management/partner/domain/exception/PartnerNotFoundException.java
package br.com.vps.consulting.b2b.management.partner.domain.exception;

import br.com.vps.consulting.b2b.management.shared.core.exception.NotFoundException;

import java.util.UUID;

public class PartnerNotFoundException extends NotFoundException {
    public PartnerNotFoundException(UUID partnerId) {
        super("Parceiro não encontrado: " + partnerId);
    }
}

File: kotlin/src/main/kotlin/br/com/vps/consulting/b2b/management/partner/domain/exception/PartnerNotFoundException.kt
package br.com.vps.consulting.b2b.management.partner.domain.exception

import br.com.vps.consulting.b2b.management.partner.domain.PartnerId
import br.com.vps.consulting.b2b.management.shared.core.exception.NotFoundException

class PartnerNotFoundException(partnerId: PartnerId) :
    NotFoundException("Parceiro não encontrado: ${partnerId.value}")
